actix-identity/src/lib.rs

```diff
@@ -843,7 +844,7 @@ mod tests {
         let cv: CookieValue = serde_json::from_str(cookie.value()).unwrap();
         assert_eq!(cv.identity, identity);
         let now = SystemTime::now();
-        let t30sec_ago = now - Duration::seconds(30).to_std().unwrap();
+        let t30sec_ago = now - std::time::Duration::try_from(Duration::seconds(30)).unwrap();
```

Comment:
Here and in a few other places in surrounding lines, `try_from` isn't necessary. Arithmetic is implemented between the time crate's values and their std equivalents. It also handles what would otherwise be overflows IIRC.

If I have missed an implementation, let me know! It's more than likely trivial to add in.